bundler/spec/bundler/bundler_spec.rb

```diff
@@ -224,24 +224,6 @@
     end
   end
 
-  describe "#rm_rf" do
-    context "the directory is world writable" do
```

Comment:
why remove the test?

Reply:
Because I cheated and you got me! 😆

This test was checking that `Bundler.rm_rf` raises a custom message. However, I'd now removed the re-raise, so keeping the test felt like testing `FileUtils`, not `Bundler`.

However, I should've added a new test that checks for the new error. I will work on it!